core/controllers/creator_dashboard.py

```diff
@@ -233,15 +233,25 @@ def _round_average_ratings(rating):
 
             subscribers_list.append(subscriber_summary)
 
+        user_settings = user_services.get_user_settings(
+            self.user_id, strict=False)
+        display_preference = user_settings.creator_dashboard_display
+
         self.values.update({
             'explorations_list': exp_summary_dicts,
             'collections_list': collection_summary_dicts,
             'dashboard_stats': dashboard_stats,
             'last_week_stats': last_week_stats,
-            'subscribers_list': subscribers_list
+            'subscribers_list': subscribers_list,
+            'display_preference': display_preference,
         })
         self.render_json(self.values)
 
+    def post(self):
+        user_creator_dashboard_display = self.payload.get('display_preference')
```

Comment:
Here, maybe user_creator_dashboard_display --> creator_dashboard_display_preference (or pref)

and below: update_creator_dashboard_display_preference(...)

Reply:
Done